src/components/molecules/author-information-list/author-information-list.tsx

```diff
@@ -32,11 +33,12 @@ const AuthorInformation = ({ author }: { author: Author }) => {
   );
 };
 
-export const AuthorInformationList = ({ authors }: { authors: Author[] }) => (
+export const AuthorInformationList = ({ authors, license, year }: { authors: Author[], license?: string, year?: number }) => (
```

Comment:
Will there be a case of there not being a year or are we assuming if there is not a year then we put the current year?

Reply:
The only reason is if there is always a year then this should be mandatory